base_classes/NXxrayedge.nxdl.xml

```diff
@@ -0,0 +1,191 @@
+      <item value="Th"> <doc>Z=90, name="thorium", standard_atomic_weight=232.038</doc> </item>
+      <item value="Pa"> <doc>Z=91, name="protactinium", standard_atomic_weight=231.036</doc> </item>
+      <item value="U"> <doc>Z=92, name="uranium", standard_atomic_weight=238.029</doc> </item>
+      <item value="Np"> <doc>Z=93, name="neptunium", standard_atomic_weight=237.048</doc> </item>
+      <item value="Pu"> <doc>Z=94, name="plutonium", standard_atomic_weight=239.052</doc> </item>
+      <item value="Am"> <doc>Z=95, name="americium", standard_atomic_weight=243.0</doc> </item>
+      <item value="Cm"> <doc>Z=96, name="curium", standard_atomic_weight=247.0</doc> </item>
+      <item value="Bk"> <doc>Z=97, name="berkelium", standard_atomic_weight=247.0</doc> </item>
+      <item value="Cf"> <doc>Z=98, name="californium", standard_atomic_weight=251.0</doc> </item>
+      <item value="Es"> <doc>Z=99, name="einsteinium", standard_atomic_weight=252</doc> </item>
+      <item value="Fm"> <doc>Z=100, name="fermium", standard_atomic_weight=257</doc> </item>
+      <item value="Md"> <doc>Z=101, name="mendelevium", standard_atomic_weight=258</doc> </item>
+      <item value="No"> <doc>Z=102, name="nobelium", standard_atomic_weight=259</doc> </item>
+      <item value="Lr"> <doc>Z=103, name="lawrencium", standard_atomic_weight=266</doc> </item>
+      <item value="Rf"> <doc>Z=104, name="rutherfordium", standard_atomic_weight=267</doc> </item>
+      <item value="Db"> <doc>Z=105, name="dubnium", standard_atomic_weight=268</doc> </item>
+      <item value="Sg"> <doc>Z=106, name="seaborgium", standard_atomic_weight=269</doc> </item>
+      <item value="Bh"> <doc>Z=107, name="bohrium", standard_atomic_weight=270</doc> </item>
+      <item value="Hs"> <doc>Z=108, name="hassium", standard_atomic_weight=269</doc> </item>
+      <item value="Mt"> <doc>Z=109, name="meitnerium", standard_atomic_weight=278</doc> </item>
+      <item value="Ds"> <doc>Z=110, name="darmstadtium", standard_atomic_weight=281</doc> </item>
+      <item value="Rg"> <doc>Z=111, name="roentgenium", standard_atomic_weight=282</doc> </item>
+      <item value="Cn"> <doc>Z=112, name="copernicium", standard_atomic_weight=285</doc> </item>
+      <item value="Nh"> <doc>Z=113, name="nihonium", standard_atomic_weight=286</doc> </item>
+      <item value="Fl"> <doc>Z=114, name="flerovium", standard_atomic_weight=289</doc> </item>
+      <item value="Mc"> <doc>Z=115, name="moscovium", standard_atomic_weight=290</doc> </item>
+      <item value="Lv"> <doc>Z=116, name="livermorium", standard_atomic_weight=293</doc> </item>
+      <item value="Ts"> <doc>Z=117, name="tennessine", standard_atomic_weight=294</doc> </item>
+      <item value="Og"> <doc>Z=118, name="oganesson", standard_atomic_weight=294</doc> </item>
+    </enumeration>
+  </field>
+  <field name="edge" type="NX_CHAR" minOccurs="1" maxOccurs="1">
+    <doc>
+      IUPAC symbol of the electronic level.
```

Comment:
[IUPAC](http://publications.iupac.org/pac/1991/pdf/6305x0735.pdf) calls these "X-ray diagram levels".

Reply:
I believe that "diagram levels" refers to the removal of the electron.  It might be a bit pedantic, but when we say "K level" we mean that ground-state level (we do not mean the K or L3 edge of an excited atom, for example -- that's a different thing, though maybe related).  I am OK with calling this the X-ayr level to follow IUPACs nomenclature.